src/components/LatestDataTable.tsx

```diff
@@ -180,7 +210,7 @@ function TxnWalletInfoDisplay({
   return (
     <>
       <div className="flex pt-5 md:pt-0 lg:w-48">
-        <span className="pr-1">From: </span>
+        <span className="pr-1 lg:pr-2">From</span>
```

Comment:
Comparing with other tables we should have space of 4px between label and value, Also can we use margin in place of padding here?

Reply:
Sure will align the space value to 4px. Noob question tho, why do we want to use margin instead of padding in this case?

Reply:
we generally use margins when you're adjusting the spacing of an element in relation to another element and we use padding for adjusting the spacing  of individual element